CONTRIBUTING.md

```diff
@@ -22,26 +22,25 @@ Please see [https://git.k8s.io/community/CLA.md](https://git.k8s.io/community/CL
 
 ### Pre-requisites
 
-Before you get started:
-
-- Install Go `1.12.3` or later
-- [Install Kubebuilder](https://book.kubebuilder.io/getting_started/installation_and_setup.html)
-- Kubernetes Cluster `1.13` or later (e.g. [Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/))
-- [Configure kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) 
+- Git
+- Go `1.12.3` or later
+- [Kubebuilder](https://book.kubebuilder.io/getting_started/installation_and_setup.html)
+- A Kubernetes Cluster running version `1.13` or later (e.g., [Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/))
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
 
 ### Build Instructions
 
-- Get KUDO repo: `go get github.com/kudobuilder/kudo/`
+- Get the KUDO repo: `go get github.com/kudobuilder/kudo/` (**Note:** KUDO uses Go Modules. Due to the current state of code generation in [controller-tools](https://github.com/kubernetes-sigs/controller-tools) and [code-generator](https://github.com/kubernetes/code-generator), KUDO currently **must** be cloned into its `$GOPATH`-based location)
 - `cd $GOPATH/src/github.com/kudobuilder/kudo`
 - `make all` to build project
-- [optionally] `make docker-build` to build docker images and `make docker-push` to push images
+- [optionally] `make docker-build` to build Docker images and `make docker-push` to push images
```

Comment:
I don't imagine most developers will be pushing to the default docker repo.  Should we include instructions on how/where to change that?

Reply:
Alternately, remove it entirely. We're trying to ship a distribution, if someone wants to build their own...that's fine too, but the common user we'd expect to be using in a private environment this _should_ be pulling a distribution image, re-tagging it, and pushing it to whatever their private distribution.

We should have separate "developing against a real cluster" instructions if those aren't covered sufficiently by `make run`.

Reply:
I left only the command to build the images, but not the one to push them.

BTW does that command build more than one image?